src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -374,6 +378,19 @@ private static Map<Project, LockedConfigurations> wireUpLockedConfigurationsByPr
                     subproject,
                     subproject.getConfigurations().getByName(CONSISTENT_VERSIONS_TEST),
                     lockedConfigurations.testConfigurations());
+
+            @SuppressWarnings("for-rollout:ConfigurationAvoidanceRegistration")
```

Comment:
Is this just because it's only recently that `.extendsFrom` takes a `Provider<Configuration>` and we still need support old Gradle? If so, it might still be better to make this a `register` than call `.get()`, and it will be easier for an errorprone to clean up later.

Reply:
This was just moved from below, so hadn't even thought about it. The `extendsFrom` is actually just in Gradle 9, so we can't even compile against it yet.

But yeah, can do the register and get!